cookbooks/travis_build_environment/recipes/pip.rb

```diff
@@ -9,7 +9,7 @@
 bash 'install-pip' do
   cwd Chef::Config[:file_cache_path]
   code <<-INSTALL_PIP
-    python get-pip.py
+    python3 get-pip.py
```

Comment:
Yes, I support that, as per https://github.com/travis-ci/travis-cookbooks/pull/1076#pullrequestreview-395767660.

My tests show that something like this should work and create `pip3`, `pip2` and `pip`, all with shebangs referencing corresponding `pythonX`s.

```suggestion
    python3 get-pip.py
    python2 get-pip.py
    python get-pip.py
```